### DISCUSSION TITLE
How to add app icons and change app name globally

### DISCUSSION THREAD

**Discussion by SomeDevWeb** [infinitered/ignite#1869] [Q&A] [answered]
Hi all,
I created a basic app successfully, for which I am very happy and deeply thankful for Ignite.
Now I need 2 things that I didn't fully found a solution for.
1. Change/edit the app name that appears in the GPlay and when it is installed.
  - I read on some forum that you can change it in the app.json, which I tried, but it still packaged it with the old name.
2. Add some icons - both normal and rounded version.
  -  Again I found in some places that you need to change the icons in the **/res/drawable-...**  folders, which I couldn't find. 
  - Instead, I found multiple mipmap-* folders with similar icons with 2 versions (square and rounded) and I replaced those ones which actually added icons to the apk file, but when I installed it on the phone, all icons were square where there was a rounded placeholder and actually in half the menus there was the default icon instead of the changes one.
  - So, can you kindly please clarify a bit where is the place in the code to change these aspects. 
  Thank you very much!!!
  Best regards, 
  Andy

**Comment by jamonholmgren** [comment]
You can change the app's name in the Google Play console and change icons in Android Studio. We could probably add more info on how to do that later.

You mentioned `app.json` -- are you using Expo? You'd want to check with Expo's docs to see how to do it there.

**Comment by flexbox** [comment, marked as answer]
Hey @SomeDevWeb 

Have you [tried to use react-native-rename](https://github.com/junedomingo/react-native-rename)

Otherwise, here is an article I have on my bookmarks 
https://saumya.github.io/ray/articles/72/

Good luck! I have done it in the past and it's not easy
